src/migrations.js

```diff
@@ -24,7 +21,7 @@ require.scopes.migrations = (function() {
 var exports = {};
 exports.Migrations= {
   changePrivacySettings: function() {
-    if (!chrome.extension.inIncognitoContext) {
+    if (!chrome.extension.inIncognitoContext && chrome.privacy ) {
```

Comment:
Currently, I'm checking for `inIncognitoContext` in a content script, then sending this as a message to be added to `pb.tabData[tab.id].inIncognito`. This is working in Chrome, but not still not firefox :/